### ISSUE TITLE
int and float

### ISSUE THREAD

**Issue by BolinLi-s** [TAMU-VITA/ShapeMatchingGAN#1]
class GlyphGenerator(nn.Module):
    def __init__(self, ngf=32, n_layers = 5):
        super(GlyphGenerator, self).__init__()
        
        encoder = []
        encoder.append(ReplicationPad2d(padding=4))
        encoder.append(Conv2d(out_channels=ngf, kernel_size=9, padding=0, in_channels=3))
        encoder.append(LeakyReLU(0.2))
        encoder.append(myGConv(ngf*2, 2, ngf))
        encoder.append(myGConv(ngf*4, 2, ngf*2))

        transformer = []
        print(n_layers/2-1)
        for n in range(n_layers/2-1):
            transformer.append(myGCombineBlock(ngf*4,p=0.0))
        # dropout to make model more robust    
        transformer.append(myGCombineBlock(ngf*4,p=0.5))
        transformer.append(myGCombineBlock(ngf*4,p=0.5))
        for n in range(n_layers/2+1,n_layers):
            transformer.append(myGCombineBlock(ngf*4,p=0.0))  

some float in range() are encontered

**Comment by williamyang1991** [comment]
> class GlyphGenerator(nn.Module):
> def **init**(self, ngf=32, n_layers = 5):
> super(GlyphGenerator, self).**init**()
> 
> ```
>     encoder = []
>     encoder.append(ReplicationPad2d(padding=4))
>     encoder.append(Conv2d(out_channels=ngf, kernel_size=9, padding=0, in_channels=3))
>     encoder.append(LeakyReLU(0.2))
>     encoder.append(myGConv(ngf*2, 2, ngf))
>     encoder.append(myGConv(ngf*4, 2, ngf*2))
> 
>     transformer = []
>     print(n_layers/2-1)
>     for n in range(n_layers/2-1):
>         transformer.append(myGCombineBlock(ngf*4,p=0.0))
>     # dropout to make model more robust    
>     transformer.append(myGCombineBlock(ngf*4,p=0.5))
>     transformer.append(myGCombineBlock(ngf*4,p=0.5))
>     for n in range(n_layers/2+1,n_layers):
>         transformer.append(myGCombineBlock(ngf*4,p=0.0))  
> ```
> 
> some float in range() are encontered

Thank you for pointing out the bug :)
It has been fixed.
